WordPress/src/main/java/org/wordpress/android/ui/stories/StoriesMediaPickerResultHandler.kt

```diff
@@ -0,0 +1,71 @@
+package org.wordpress.android.ui.stories
+
+import android.app.Activity
+import android.content.Intent
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.ui.ActivityLauncher
+import org.wordpress.android.ui.PagePostCreationSourcesDetail
+import org.wordpress.android.ui.media.MediaBrowserActivity
+import org.wordpress.android.ui.media.MediaBrowserType
+import org.wordpress.android.ui.photopicker.PhotoPickerActivity
+import org.wordpress.android.util.AppLog
+import org.wordpress.android.util.AppLog.T.UTILS
+
+class StoriesMediaPickerResultHandler {
+    companion object {
+        /* return true if MediaPickerResult was handled */
+        fun handleMediaPickerResultForStories(
+            data: Intent,
+            activity: Activity?,
+            selectedSite: SiteModel?
```

Comment:
Since these are nullable should I do a check to ensure if either or null we don't continue here? The original logic did not check for this so I proceeded as is but I am now just checking with you to get a clearer understanding of how you view this. LMK 🚢

Reply:
Good question - I think it's fine given there are other checks elsewhere that would prevent this from happening here. We could requireNotNull() but given we're not handling errors at this point I think it's ok to let them be handled elsewhere 👍

Reply:
Okay thanks much!